### ISSUE TITLE
Display ID is not always :0 after a reboot

### ISSUE THREAD

**Issue by aethernet** [balena-io-playground/xserver#13]
Sometimes DISPLAY is :0, sometimes :1, sometimes higher.

Seems to be related to reboots of the machine, but might also occurs on service restart.
It doesn't happen every time so it's hard to pin down the conditions.

Tested on x86 with service installed from dockerhub.

``` 
xserver:
    image: balenablocks/xserver
    restart: always
    privileged: true
    volumes:
      - "x11:/tmp/.X11-unix"
    environment:
      - CURSOR=false
```

Note that there's always only one entry in `/tmp/.X11-unix/` but not always `X0`

My current workaround is to add this to my service start script : 

```
while [ `ls /tmp/.X11-unix/ | wc -l` == 0 ]; do echo "display not created yet"; sleep 0.5; done; echo "display created";
DISPLAY=`ls /tmp/.X11-unix/ | sed s/X/:/`
export DISPLAY
echo "DISPLAY is set to $DISPLAY"
while ! xset -q; do echo "display doesn't accept connection yet"; sleep 0.5; done; echo "display ready";
```

**Comment by mvheimburg** [comment]
I had the same issue. Thank you for this workaround :)

**Comment by nucleardreamer** [comment]
This should be resolved by #15 now. The default display is set to `:0` and you can override however you like!
